examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -172,14 +146,16 @@ async def wait_for_done():
                         # until all samples are read.
                         if done_response.status:
                             every_n_samples_stream.cancel()
-                        await raise_if_error(done_response)
                 except asyncio.CancelledError:
                     pass
 
             await asyncio.gather(read_data(), wait_for_done())
 
         except grpc.RpcError as rpc_error:
             error_message = rpc_error.details()
+            for key, value in rpc_error.trailing_metadata() or []:
+                if key == "ni-error":
+                    error_message += f"\nError status: {value}"
```

Comment:
Note that this likely needs to change. I've made some changes since my original post and I don't think it's final yet. Stay tuned.

Reply:
I've updated the review so that every file's exception-handling uses the code block that you ended using for #717 (Update switch/tclk/xnet/session examples to display error code).